crud/common/call.lua

```diff
@@ -160,6 +160,8 @@ function call.rw_single(bucket_id, func_name, func_args, options)
             )
         end
 
+        err = errors.wrap(err)
```

Comment:
Is it really needed?

Reply:
Yes, error is a table with overloaded `tostring` fuction. We need to reconstruct error received from connection to correctly show it.

Reply:
But why did it work ok before?

Reply:
It didn't =)

Reply:
Schema wasn't changed in tests before this patch, so all `Space doesn't exists` errors were returned directly by router.